Fast Pair headers: new UUID header (#14224)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`bt_fast_pair_readme` library:

>   * Added:
> 
>     * The :c:func:`bt_fast_pair_enable`, :c:func:`bt_fast_pair_disable`, :c:func:`bt_fast_pair_is_ready` functions to handle the Fast Pair subsystem readiness.
>     In order to use the Fast Pair service, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
>     * The :file:`ncs/nrf/include/bluetooth/services/fast_pair/uuid.h` header file that contains UUID definitions for the Fast Pair service and its characteristics.

Reviewer: ```suggestion
  * Added:

    * The :c:func:`bt_fast_pair_enable`, :c:func:`bt_fast_pair_disable`, :c:func:`bt_fast_pair_is_ready` functions to handle the Fast Pair subsystem readiness.
      In order to use the Fast Pair service, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
    * The :file:`include/bluetooth/services/fast_pair/uuid.h` header file that contains UUID definitions for the Fast Pair service and its characteristics.
```

Reviewer: This fixes the doc build error.

  * Updated:

    * Improved the :ref:`bt_fast_pair_readme` library documentation to include the description of the missing Kconfig options.

>     * The location of the :file:`ncs/nrf/include/bluetooth/services/fast_pair.h` header file and moved it to the :file:`ncs/nrf/include/bluetooth/services/fast_pair/` directory to have a common place for Fast Pair headers.

Reviewer: ```suggestion
    * The :file:`include/bluetooth/services/fast_pair.h` header file is moved to the :file:`ncs/nrf/include/bluetooth/services/fast_pair` directory to have a common place for Fast Pair headers.
```
